packages/theme-ui/src/index.ts

```diff
@@ -17,7 +17,6 @@ export * from '@theme-ui/components'
 export {
   css,
   get,
-  ColorMode,
```

Comment:
I *think* this might be a TypeScript type, but not sure. Can anyone clarify? cc @mxstbr

Reply:
yup, if you hover it in a text editor, this ugly thing shows up

![image](https://user-images.githubusercontent.com/15332326/81442808-8c0b3500-9174-11ea-83c5-910dd443dc40.png)

and if you scroll down, there's a comment which is a bit too long and outdated. It actually references the old ColorMode component

![image](https://user-images.githubusercontent.com/15332326/81442928-c674d200-9174-11ea-8a1d-73fa6de0fcce.png)

I'll give it some more attention and update it.